devices/casa_cmts.py

```diff
@@ -446,6 +447,26 @@ def check_docsis_mac_ip_provisioning_mode(self, index):
         if self.match != None:
             return result
 
+    def get_ertr_ipv4(self, mac):
+        self.sendline("show cable modem %s cpe" % mac)
+        self.expect(self.prompt)
+        ertr_ipv4 = re.search('(%s) .*(eRouter)'% ValidIpv4AddressRegex ,self.before)
+        if ertr_ipv4:
+            ipv4 =ertr_ipv4.group(0)
+            return ipv4
+        else:
+            return None
+
+    def get_ertr_ipv6(self, mac):
+        self.sendline("show cable modem %s cpe" % mac)
+        self.expect(self.prompt)
+        ertr_ipv6 = re.search(ValidIpv6AddressRegex ,self.before)
+        if ertr_ipv6:
+            ipv6 = ertr_ipv6.group()
+            return ipv6
+        else:
+            return None
```

Comment:
I don't want to keep beating a dead horse, but should these really return None or raise an Exception?

I will merge as-is for now, but please make sure this is thought out properly.